### ISSUE TITLE
adaptive learning rate

### ISSUE THREAD

**Issue by harell** [data-science-competitions/Modeling-Earthquake-Damage#31]
Figure out how to change the learning rate (`eta`) during XGBoost execution.

Hint: see cb.rest.parameters + call back function in XGBoost.train
